aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java

```diff
@@ -281,6 +281,7 @@ protected String defaultWarehouseLocation(TableIdentifier tableIdentifier) {
                 .build());
     String dbLocationUri = response.database().locationUri();
     if (dbLocationUri != null) {
+      dbLocationUri = LocationUtil.stripTrailingSlash(dbLocationUri);
```

Comment:
looks like same response was used in method `loadNamespaceMetadata` (Line 518) as well, do we need to stripTrail there as well?

Reply:
I agree, I think we'd need to do the same in `loadNamespaceMetadata()`